src/compiler/core.ts

```diff
@@ -2279,14 +2280,21 @@ namespace ts {
         this.original = undefined;
     }
 
+    function SourceMapSource(this: SourceMapSource, fileName: string, text: string, skipTrivia?: (pos: number) => number) {
+        this.fileName = fileName;
+        this.text = text;
+        if (skipTrivia) this.skipTrivia = skipTrivia;

```

Comment:
You lost the default for `skipTrivia` with this change. Previously you had `pos => pos`. Was this intended?